sys/aarch64/pmap.c

```diff
@@ -602,3 +604,25 @@ void pmap_delete(pmap_t *pmap) {
   free_asid(pmap->asid);
   pool_free(P_PMAP, pmap);
 }
+
+/*
+ * Increase usable kernel virtual address space to at least maxkvaddr.
+ * Allocate page table (level 1) if needed.
+ */
+vaddr_t pmap_growkernel(vaddr_t maxkvaddr) {
+  pmap_t *pmap = pmap_kernel();
+  vaddr_t va;
+
+  /* For 8 pages used by kernel we need 1 page for KASAN. */
+  maxkvaddr = roundup(maxkvaddr, 8 * L2_SIZE);
```

Comment:
The `roundup()` here is so that the shadow map is grown by a multiple of `L2_SIZE`, is my understanding correct? If so, I'd like the comment to say it more directly.

Reply:
done